docs/pages/2020_Intro_Week/sections/running.rst

```diff
@@ -294,7 +294,7 @@ Define a set of input parameters for Quantum ESPRESSO, preparing a dictionary of
        ...:     'SYSTEM': {
        ...:         'ecutwfc': 30.,
        ...:         'ecutrho': 200.,
-       ...:         'mickeymouse': 240.,
+       ...:         'mickeymouseouse': 240.,
```

Comment:
mickeymouseouse?

Reply:
Bad search and replace - don't know what happened.